packages/next/src/views/List/renderListViewSlots.tsx

```diff
@@ -44,6 +44,18 @@ export const renderListViewSlots = ({
     })
   }
 
+  const listMenuItems = collectionConfig.admin.components?.listMenuItems
+  if (Array.isArray(listMenuItems)) {
+    result.listMenuItems = [
+      RenderServerComponent({
+        clientProps,
+        Component: listMenuItems,
+        importMap: payload.importMap,
+        serverProps,
```

Comment:
These props are left untyped. See the latest pattern using the `satisfies` keyword. Each component needs its own set of explicit types not just to make it type-safe, but also to make it as easy as possible for component authors.